Scenarios/settings.md

```diff
@@ -56,6 +56,7 @@ The repository settings are only read from the repository settings file (.github
 | <a id="CICDSchedule"></a>CICDSchedule | CRON schedule for when CI/CD workflow should run. Default is no scheduled run, only manually triggered or triggered by Push or Pull Request. Build your CRON string here: [https://crontab.guru](https://crontab.guru) |
 | <a id="UpdateGitHubGoSystemFilesSchedule"></a>UpdateGitHubGoSystemFilesSchedule | CRON schedule for when Update AL-Go System Files should run. When Update AL-Go System Files runs on a schedule, it uses direct COMMIT instead of creating a PR. Default is no scheduled run, only manual trigger. Build your CRON string here: [https://crontab.guru](https://crontab.guru) |
 | <a id="buildModes"></a>buildModes | A list of build modes to use when building the AL-Go projects. Every AL-Go projects will be built using each built mode. Available build modes are:<br /> **Default**: Apps are compiled as they are in the source code.<br />**Clean**: _PreprocessorSymbols_ are enabled when compiling the apps. The values for the symbols correspond to the `cleanModePreprocessorSymbols` setting of the AL-Go project.<br />**Translated**: `TranslationFile` compiler feature is enabled when compiling the apps. |
+| <a id="SecretlessPRBuild"></a>SecretlessPRBuild | Set to true if PR Builds should be triggered by the [pull_request](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request) event. When set to true, workflows triggered from forked repositories will not be able to access secrets with the exception of GITHUB_TOKEN |
```

Comment:
Just an opinion: let's call the setting `PRBuildTrigger` with options `pull_request` and `pull_request_target`(default).
Mainly to handle the case if GitHub adds support for a third value.